Mathlib/Topology/Sets/Closeds.lean

```diff
@@ -105,7 +106,7 @@ instance : Inhabited (Closeds α) :=
   ⟨⊥⟩
 
 @[simp, norm_cast]
-theorem coe_sup (s t : Closeds α) : (↑(s ⊔ t) : Set α) = ↑s ∪ ↑t :=
+theorem coe_sup (s t : Closeds α) : (↑(s ⊔ t) : Set α) = ↑s ∪ ↑t := by
```

Comment:
why did you change this one?

Reply:
I can revert this, but I needed to see the statement in the infoview to click on the types of things to ensure that the statement said the correct thing. It is annoying that this is only possible in tactic mode.